src/codegen/templates/constants.mako

```diff
@@ -0,0 +1,26 @@
+<%
+import utilities.enum_helpers as enum_helpers
```

Comment:
`utilities` is a submodule of `codegen`.
```suggestion
import codegen.utilities.enum_helpers as enum_helpers
```

Reply:
Resolved.